docs/components/modeler/desktop-modeler/flags/flags.md

```diff
@@ -43,6 +43,10 @@ Flags passed as command line arguments take precedence over those configured via
 
 ## Examples
 
+### Disable Plug-ins
```

Comment:
```suggestion
### Disable Plugins
```

Or we could change the link on line 31: 
```diff
-| ["disable-plugins"](#disable-plugins)
+| ["disable-plugins"](#disable-plug-ins)
```

Reply:
The later is the way to go, I guess. Let's aim for human readable sections.

Reply:
Fixed via https://github.com/camunda/camunda-platform-docs/pull/2027/commits/16addb5dbadd9810ab4e8f73413cd28bf6652585.